Source/Core/Core/PowerPC/GDBStub.cpp

```diff
@@ -480,6 +602,14 @@ static void WriteRegister()
   {
     rPS(id - 32).SetPS0(re64hex(bufptr));
   }
+  else if (id >= 71 && id < 87)
+  {
+    PowerPC::ppcState.sr[id - 71] = re32hex(bufptr);
+  }
+  else if (id >= 88 && id < 104)
+  {
+    PowerPC::ppcState.sr[SPR_IBAT0U + id - 88] = re32hex(bufptr);
```

Comment:
This is a typo, and should be `spr` instead of `sr`.  (It also generates a compile warning.)